docs/Controls.md

```diff
@@ -4,7 +4,7 @@
 
 When armed the aircraft is ready to fly and any motors attached will spin when throttle is applied.
 
-By default arming and disarming is done using stick positions.  This is disabled when using a switch to arm.
+By default arming and disarming is done using stick positions.  Alternatively, you can use a switch to arm.
```

Comment:
I don't think that saying "alternatively" here makes it clear enough that stick arming no longer works at all if an arming switch is configured.